### PR TITLE
Create Dont ask to ask.md

### PR DESCRIPTION
**Pull request by chrisgrieser**

### CODE REVIEW

File: 05 - Concepts/Dont ask to ask.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Don't ask to ask, just ask
+
```
Comment (claremacrae) [inline 05 - Concepts/Dont ask to ask.md:10]: Am trying to read this as someone who has not used this same phrase myself online...

I think people might reasonably ask why this page exists, and who it is aimed at.

The intro on [https://dontasktoask.com](https://dontasktoask.com) says:

> Every now and then, in online chat rooms I hang around in, someone pops in and says something in the lines of,

So I wonder about something like:

> Every now and then, in the Obsidian Members Group Discord server, someone pops in and says something in the lines of:

**Comment by argenos** [comment]
I thought we agreed this was off-topic for the hub and was better as a Carl message?

**Comment by chrisgrieser** [comment]
yes, but then I thought, why not both? doesn't hurt to have it in the hub, too, for the rare case of being able to refer to it outside of Discord.

File: 05 - Concepts/Dont ask to ask.md (hunk 1 above)
Comment (chrisgrieser) [reply 05 - Concepts/Dont ask to ask.md:10]: well, the dont-ask-to-ask principle does work also holds outside of the Obsidian Discord server, so I phrased it mroe generally.

**Comment by chrisgrieser** [comment]
closing this and instead making a new PR with a Discord GUIDE

**Pull request closed without merging**